actions/create-signing-events/action.yml

```diff
@@ -29,11 +35,12 @@ runs:
       shell: bash
 
     - name: Dispatch signing event workflow
-      if: steps.create-signing-events.outputs.events != ''
+      if: inputs.token == github.token && steps.create-signing-events.outputs.events != ''
```

Comment:
Here I think a comment that `GITHUB_TOKEN` does not automatically dispatch a workflow would help. It's documented in `actions/signing-event/action.yml` but that context may be missing.